.github/workflows/quality-control.yml

```diff
@@ -2,103 +2,24 @@ name: Quality Control
 
 on:
   push:
-    branches: [main]
+    branches: [ main ]
   workflow_dispatch:
 
 jobs:
-  validate-composer:
```

Comment:
remove the redundant jobs

Reply:
Let's leave these for the time being, please. These were added for the days when @brendt feels rebellious and pushes to the main branch without a PR. 😉 

I do think we can make this more efficient in the future, but not a problem to solve today.

Reply:
I see that hasn't changed :trollface: 

This could be done through branch protection rules, but he'd be able to circumvent those as he's the admin 🤣 
I'll keep the push event in

Reply:
Agreed on branch protection rules, but I'm not going to hijack someone else's project. 🙂